spec/requests/volunteers_spec.rb

```diff
@@ -171,13 +172,14 @@
         volunteer.supervisor = build(:supervisor)
 
         patch volunteer_path(volunteer), params: {
-          volunteer: {email: other_volunteer.email, display_name: "New Name"}
+          volunteer: {email: other_volunteer.email, display_name: "New Name", phone_number: "+14163218092"}
         }
         expect(response).to have_http_status(:success) # Re-renders form
 
         volunteer.reload
         expect(volunteer.display_name).to_not eq "New Name"
         expect(volunteer.email).to_not eq other_volunteer.email
+        expect(volunteer.phone_number).to_not eq "+14163218092"
```

Comment:
This looks too much like a real phone number, please replace it with an obviously fake phone number

Reply:
fixed